.github/workflows/broken-linkcheck.yml

```diff
@@ -0,0 +1,41 @@
+name: Linkcheck
+on:
+    workflow_dispatch: 
+    # Enable manual triggering
+    schedule:
+    # Run monthly on the 1st day of the month
+        - cron: '0 0 1 * *'
+    pull_request:
+      paths:
+        - ".github/workflows/broken-linkcheck.yml"
+        - "doc/make.py"
+jobs:
+    linkcheck:
+        runs-on: ubuntu-latest
+        defaults:
+          run:
+                shell: bash -el {0}
+
+        steps:
+        - name: Checkout
+          uses: actions/checkout@v4
+          with:
+                fetch-depth: 0
+
+        - name: Set up Conda
+          uses: ./.github/actions/setup-conda
+
+        - name: Build Pandas
+          uses: ./.github/actions/build_pandas
+
+        - name: Run linkcheck script
+          working-directory: ./doc
+          shell: bash
```

Comment:
Sorry, I broke this in my last suggestion. We need to use the default shell here to get access to the conda env.

```suggestion
```